rails-commenting.rb

```diff
@@ -71,6 +71,6 @@ def blog_post_params
 # FILE: app/models/blog_post.rb
 
 class BlogPost < ApplicationRecord
-  # 10)
+  # 10) This is the association. It describes the relationships between tables. 
```

Comment:
After working through the wildlife tracker this looks accurate and correct.  I appreciate the efficient descriptions. Great work!